ome/resource_network_setting.go

```diff
@@ -180,6 +216,19 @@ func (r *networkSettingResource) Update(ctx context.Context, req resource.Update
 		return
 	}
 	defer omeClient.RemoveSession()
+	// session configuration
+	if state.OmeSessionSetting != nil && plan.OmeSessionSetting != nil {
```

Comment:
If plan.OmeSessionSetting != nil but state.OmeSessionSetting == nil, then also settings shd be updated, rt?
Please check if the logic is correct.